api/api.ts

```diff
@@ -13,48 +13,88 @@ dotenvConfig()
 
 const authUrl = process.env.URL_AUTH_SERVICE;
 
+type AuthStatus = {
```

Comment:
There's a scpecific file only for types. Is it possible to put these types there ?

Reply:
done